Header header
sensor_msgs/CameraInfo camera_info
BoundingBox[] bounding_boxes
